applications/FluidDynamicsApplication/python_scripts/apply_behr_slip_process.py

```diff
@@ -0,0 +1,58 @@
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+
+def Factory(settings, Model):
+
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return BehrWallCondition(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class BehrWallCondition(KratosMultiphysics.Process):
```

Comment:
Which is the difference of this process compare against the common one? I can't see a major difference, so we can slightly modify the current one according to the Behr contribution necessities.

Reply:
I am sorry, I simply forgot to delete this file. Indeed, it is not needed after the Behr2004 contribution was integrated into NavierStokesWallConditio.